src/components/InviteForm.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { shareList } from '../api';
+import { useAuth } from '../api/useAuth.jsx';
+
+const InviteForm = ({ listPath }) => {
+	const [input, setInput] = useState({
+		recipientEmail: '',
```

Comment:
Would using just a string as the value of input make more sense here? (see next comment for more details)

Reply:
We made it this way because of the parameters for the shareList function and wanting to keep the same naming for the firebase schema. We initially had just an empty string, but it wasn't working properly. Though,  we also changed some other things. Could try it out again.

Reply:
Okay, I tested the code with these changes and it still worked. Thanks for the input!